src/partials/must_change_password.hbs

```diff
@@ -16,24 +16,24 @@
     {{/if}}
     <div class="stack stack--small">
       <div class="float-label">
-        <input type="text" class="text-input float-label__input" name="username" placeholder="Username"
+        <input type="text" class="text-input float-label__input required" name="username" placeholder="Username"
           value="{{username}}" autofocus />
         <label class="float-label__label" for="username13">Username</label>
       </div>
       <div class="stack stack--small">
         <div class="float-label">
-          <input type="password" class="text-input float-label__input" name="existingPassword"
+          <input type="password" class="text-input float-label__input required" name="existingPassword"
```

Comment:
Just for future reference, the new change password flow with policy will need this field to be optional